wpilibj/src/test/resources/edu/wpi/first/wpilibj/PreferencesTestDefault.ini

```diff
@@ -1,7 +1,7 @@
 [NetworkTables Storage 3.0]
 double "/Preferences/checkedValueInt"=2
-double "/Preferences/checkedValueDouble"=.2
-double "/Preferences/checkedValueFloat"=3.14
+double "/Preferences/checkedValueDouble"=0.2
+double "/Preferences/checkedValueFloat"=3.4
```

Comment:
Why does this test need changing?

Reply:
spotbugs warns about using a value suspiciously close to pi, so it's easy enough to just use a different number. I added a leading zero to ".2" because not including leading zeroes is against our style. If it's part of the test, that should be documented.

Reply:
I'm pretty sure when written it will include the leading zero.  So yes, it's part of the test.